wokspace-loader/src/index.html

```diff
@@ -18,13 +18,21 @@
     <% if(htmlWebpackPlugin.options.cssName) { %>
     <link href="<%= htmlWebpackPlugin.options.urlPrefix %><%= htmlWebpackPlugin.options.cssName %>" rel="stylesheet">
     <% } %>
-    <script src="<%= htmlWebpackPlugin.options.urlPrefix %><%= htmlWebpackPlugin.files.js %>" type="text/javascript"></script>
+    <script src="<%= htmlWebpackPlugin.options.urlPrefix %><%= htmlWebpackPlugin.files.js %>" type="text/javascript" async="true"></script>
 </head>
 
-<body>
-    <main>
-        <div id="output"></div>
-    </main>
+<body style="background-color: #21252b; transition: background-color 0.5s ease;">
```

Comment:
why the style is in the html and not in the css ?

Reply:
It's for prevent blinking of the background. The page has grey background and new background color is used only after css file has been loaded. I played too much with this, only moving styles inside body resolves the problem.

Reply:
I would say that the main issue is that the JS script should be loaded at the last item of body document, not in head
while css should be kept in `<head>` element